src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(number) {
+    return function(argument) {
+      count++;
```

Comment:
i would prefer to rename variable count to something like currentCallCount  to be  clear